### ISSUE TITLE
API stopped returning proper ticket information

### ISSUE THREAD

**Issue by derhuerst** [derhuerst/vbb-hafas#17]
Previously, the parsable ticket information looked as follows, [allowing you to parse the fafe zone, wether it's a group ticket and wether it's a reduced ticket](https://github.com/derhuerst/vbb-parse-ticket).

```json
{
	"name": "Regeltarif",
	"prc": 1320,
	"cur": "EUR",
	"shpCtx": "{\"FV\":\"VBB-1\",\"TC\":\"Bartarif\",\"SW\":\"5555\",\"ZW\":\"5851\",\"TLS\":\"B3M\",\"VT\":\"Berlin ABC\"}"
}
```

Now the information looks like this, letting you parse only the name and the price:

```json
{
	"name": "Regeltarif",
	"prc": 1320,
	"cur": "EUR",
	"shpCtx": "{\"FV\":\"BVG-1\",\"TN\":\"4-Fahrten-Karte Regeltarif Berlin Tarifgebiet A-B-C\",\"SL\":\"S+U Friedrichstr. Bhf (Berlin)\",\"LI\":\"9100001\",\"ZL\":[],\"TL\":\"Berlin Tarifgebiet A-B-C\",\"VB\":\"2017-05-17T14:37\"}"
}
```

**Comment by derhuerst** [comment]
In https://github.com/derhuerst/vbb-hafas/commit/43355d723fa98347a303d5c8056d788389acdb0c, i have set up logging to collect ticket info if returned by the VBB API.

**Comment by derhuerst** [comment]
fixed in https://github.com/derhuerst/vbb-hafas/commit/74680b70386d420882cb628218cbb736b341f701, released as `vbb-hafas@3.1.0`.
